### PR TITLE
Add TechNepal.io

### PR DESCRIPTION
**Pull request by nkhanal0**
Add the TechNepal portal (https://technepal.io) to the Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [SUBISU CABLENET (P.) LTD.](http://subisu.net.np/) | Jhamsikhel, Lalitpur | Internet, Network and Television Service Provider
 - [Sustainable Solutions](http://sussol.net/) | Pulchowk, Lalitpur | mSupply and MoneyWorks
 - [Tech Bizz Pvt. Ltd.](http://www.techbizznepal.com/) | Maitidevi, Kathmandu | Mobile Apps, Graphics Design, and Web Development Services
+- [TechNepal](https://technepal.io) | Online. | One stop tech news portal in Nepal.
```
Comment (mesaugat) [inline README.md]: Thank you for the contribution. Can you also add your location? @nkhanal0 

I get that you or your team might be working remotely but you might want to add in the location that you have registered or can relate to.

Comment (nkhanal0) [reply README.md]: I will do that. Thanks

**Comment by nkhanal0** [comment]
@mesaugat : I have updated my pull request by pushing to my master. It should be good now. Please let me know if it works.

**Comment by mesaugat** [comment]
@nkhanal0 This list is specifically for companies based in Nepal. Do you have a subsidy or a working location in Nepal?

**Comment by nkhanal0** [comment]
@mesaugat : I have updated it to reflect the correct working location.

**Pull request merged**